astro/src/components/api/API.astro

```diff
@@ -24,90 +31,113 @@ if (uri) {
   let q = uri.indexOf("?");
   if (q >= 0) {
     parameters = uri.substring(q);
-    url = url.substring(0, q );
+    url = url.substring(0, q);
   }
 
   // Optionally extract one or more segment.
-  let type = 'literal';
-  let atom = '';
+  let type = "literal";
+  let atom = "";
 
   for (let i = 0; i < url.length; i++) {
     let c = url.charAt(i);
     atom += c;
 
-    if (c === '{') {
+    if (c === "{") {
       if (atom.length > 0) {
-        parts.push({type: 'literal', value: atom.substring(0, atom.length - 1)});
+        parts.push({
+          type: "literal",
+          value: atom.substring(0, atom.length - 1),
+        });
       }
-      type = 'segment';
-      atom = '{';
-    } else if (c === '}') {
-      parts.push({type: 'segment', value: atom});
-      type = 'literal';
-      atom = '';
+      type = "segment";
+      atom = "{";
+    } else if (c === "}") {
+      parts.push({ type: "segment", value: atom });
+      type = "literal";
+      atom = "";
     }
   }
 
   if (atom.length > 0) {
-    parts.push({type: 'literal', value: atom});
+    parts.push({ type: "literal", value: atom });
   }
 
   // Account for a literal URL w/out any parameter segments
   if (parts.length === 0) {
-    parts.push({type: 'literal', value: url});
+    parts.push({ type: "literal", value: url });
   }
 }
 
 const authenticationTypes = {
-  'api-key': '/docs/apis/authentication#api-key-authentication',
-  'none': '/docs/apis/authentication#no-authentication-required',
-  'jwt': '/docs/apis/authentication#jwt-authentication',
-  'basic': '/docs/apis/authentication#basic-authentication-using-an-api-key',
-  'client-credentials': '/docs/apis/authentication#client-credentials',
-  'local-bypass': '/docs/apis/authentication#localhost-authentication-bypass'
-}
+  "api-key": "/docs/apis/authentication#api-key-authentication",
+  none: "/docs/apis/authentication#no-authentication-required",
+  jwt: "/docs/apis/authentication#jwt-authentication",
+  basic: "/docs/apis/authentication#basic-authentication-using-an-api-key",
+  "client-credentials": "/docs/apis/authentication#client-credentials",
+  "local-bypass": "/docs/apis/authentication#localhost-authentication-bypass",
+};
 ---
 
-{(authentication || title) &&
-<div class="border-b border-slate-300 dark:border-slate-800 flex flex-row align-middle">
-  { authentication &&
-    <div class="not-prose flex mb-2 flex-row mx-1">
-      { authentication.map(auth =>
-          <a class="no-underline flex" href={authenticationTypes[auth]}>
-            <APIAuthenticationIcon type={auth}/>
-          </a>
+{
+  (authentication || title) && (
```

Comment:
Yeah the formatting changes are pretty distracting and it is hard to tell what is content and what is syntax